cristin-import/src/test/resources/features/EventRules.feature

```diff
@@ -29,3 +29,23 @@ Feature: Event conversion rules
       | POPVIT_FOREDRAG   | Lecture           |
       | ANNEN_PRESENTASJ  | OtherPresentation |
       | UTST_WEB          | OtherPresentation |
+
+
+  Scenario: Cristin Result of type foredrag is converted to NVA Resource
+  with Publication Context of type "Event"
+    Given a valid Cristin Result with secondary category "VIT_FOREDRAG"
+    And the Cristin Result has a non empty LectureOrPoster
+    And the LectureOrPoster has an Event
+    And the PresentationEvent has a title "Some label"
+    And the PresentationEvent has an Agent "Some agent"
+    And the PresentationEvent has a country code "NO"
+    And the PresentationEvent has a place "Norge"
+    And the PresentationEvent has a from date "2023-11-28T00:00:00"
+    And the PresentationEvent has a to date "2023-11-29T00:00:00"
```

Comment:
Vært finere med en type registry, men greit nok, den gjør det samme bare med masse ekstra gherkin syntax.

Her er offisiell dokumentasjon om hvordan man gjør det: https://cucumber.io/docs/cucumber/configuration/?lang=java

Og her er to lenker på hvordan vi har gjort det i Cristin-import:
https://github.com/BIBSYSDEV/nva-publication-api/blob/main/cristin-import/src/test/java/cucumber/utils/transformers/ArtisticEventTransformer.java

Og slik blir det skrevet i gherkin:
`    And the performance has an event with values:
      | start               | title                                           | place     | end                 |
      | 2018-02-03T00:00:00 | Celebratory Concert of Swiss Philosophy Society | Göttingen | 2020-02-03T00:00:00 |`